src/docs/databases/mongodb.md

```diff
@@ -7,13 +7,21 @@ MongoDB database with zero configuration.
 
 ## Connect
 
-When you run `railway run` in a project with the MongoDB database service installed, we inject several environment variables.
+There are two ways to connect to a MongoDB database:
 
-- MONGOHOST
-- MONGOPORT
-- MONGOUSER
-- MONGOPASSWORD
-- MONGO_URL
+- Add a [Reference Variable](/develop/variables#reference-variables) to a service
+- Run `railway connect` to start a `mongodb` shell
+
+The following variables are made available to reference in your services:
```

Comment:
We should disambiguate the ways on how people connect to the DB. We should make it explicit that exposing the variable to the service will give the service access to the DB.